tests/unit/lms/security_test.py

```diff
@@ -226,9 +214,31 @@ def test_permits_denied(self, pyramid_request):
 
         assert is_allowed == Denied("denied")
 
-    @pytest.fixture(autouse=True)
-    def _authenticated_userid(self, patch):
-        return patch("lms.security._authenticated_userid")
+    @pytest.mark.parametrize(
+        "lti_user,expected_userid",
+        [
+            (None, None),
+            (
+                factories.LTIUser(
+                    user_id="sam",
+                    application_instance_id=100,
+                ),
+                "c2Ft:100",
+            ),
+            (
+                factories.LTIUser(
+                    user_id="Sam:Smith",
+                    application_instance_id=200,
+                ),
+                "U2FtOlNtaXRo:200",
+            ),
+        ],
```

Comment:
These two last cases look like the same case to me?

Reply:
Yes, leftover from before. Removed one.